packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testTruncate.st

```diff
@@ -0,0 +1,9 @@
+testing
+testTruncate
+
+	| scrollPane |
+	instance bounds: (50 @ 50 corner: 200 @ 200).
+	scrollPane := SScrollPane forMorph: instance.
+	instance truncateDiagram.
+
+	self assert: (instance boundsInWorld containsRect: scrollPane boundsInWorld).
```

Comment:
what gets tested here? That the diagram gets truncated to the size of the scrollpane if no diagrams are on it?